PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+namespace Builder
+{
+    public interface IPizzaBuilder
+    {
+        void BuildDough();
+        void BuildSauce();
+        void BuildTopping();
+    }
+    public class WhitePizzaBuilder : IPizzaBuilder
+    {
+        private Pizza _pizza = new Pizza();
+        public WhitePizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this._pizza = new Pizza();
+        }
+        public void BuildDough()
+        {
+            this._pizza.Add("Dough");
+        }
+        public void BuildSauce()
+        {
+            this._pizza.Add("Sausce");
+        }
+        public void BuildTopping()
+        {
+            this._pizza.Add("Topping");
+        }
+        public Pizza getPizza()
+        {
+            Pizza result = this._pizza;
+            this.Reset();
+            return result;
+        }
+    }
+    public class WheatPizzaBuilder : IPizzaBuilder
+    {
+        private Pizza _pizza = new Pizza();
+        public WheatPizzaBuilder()
+        {
+            this.Reset();
+        }
+        public void Reset()
+        {
+            this._pizza = new Pizza();
+        }
+        public void BuildDough()
+        {
+            this._pizza.Add("Dough");
+        }
+        public void BuildSauce()
+        {
+            this._pizza.Add("Sausce");
+        }
+        public void BuildTopping()
+        {
+            this._pizza.Add("Topping");
+        }
+        public Pizza getPizza()
+        {
+            Pizza result = this._pizza;
+            this.Reset();
+            return result;
+        }
+    }
+    public class Pizza
```

Comment:
Good job declaring the `pizza` class but even though your solution is correct, it should have been two classes as written in the requirements.